tests/e2e/sensor.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091
+
+set -euo pipefail
+
+# Runs all e2e tests. Derived from the workload of CircleCI gke-api-nongroovy-tests.
```

Comment:
```suggestion
# Runs sensor integration tests.
```

Reply:
I will merge now to unblock [the other PR](https://github.com/openshift/release/pull/40182), but I will correct this comment in the followup PR.

Reply:
Here it is: https://github.com/stackrox/stackrox/pull/6416